src/modals/manage-farming-pool-modal/stake-form/use-farm-tokens.ts

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react';
+
+import { SingleFarmResponse } from 'src/apis/quipuswap-staking/types';
+import { useTokenExchangeRateGetter } from 'src/hooks/use-token-exchange-rate-getter.hook';
+import {
+  useAssetsListSelector,
+  useSelectedAccountSelector,
+  useTezosTokenSelector
+} from 'src/store/wallet/wallet-selectors';
+import { TEZ_TOKEN_SLUG } from 'src/token/data/tokens-metadata';
+import { toTokenSlug } from 'src/token/utils/token.utils';
+import { isDefined } from 'src/utils/is-defined';
+import { convertFarmToken } from 'src/utils/staking.utils';
+
+export const useFarmTokens = (farm?: SingleFarmResponse) => {
```

Comment:
This function looks like an incomplete copy of one with the same name from `src/hooks/use-farm-tokens.ts`. Let's leave only one of them